.gitattributes

```
@@ -1,5 +1,5 @@
 # Auto detect text files and perform LF normalization
-* text=off
+* text=auto
```

```suggestion
* text=auto
* text eol=lf
```
Looks like ideally, we should be further explicit.  This seems to help especially, and when `eol` is unspecified, it [ends up converting to CRLF on Windows by default](https://git-scm.com/docs/gitattributes#_eol), and just not really ideal for our cross platform development, so ends up causing more issues overall.  It mitigate that somewhat for Windows users especially, let's also set `eol=lf` and that will help since it's highly likely that folks typically will NOT have set their `core.autocrlf=true` or `core.autocrlf=input`.

Poking around, this seems to be the settings that other open source projects do that are cross platform, they explicitly normalize the repo to LF endings.